npm-packages/meteor-babel/package.json

```
@@ -47,7 +47,7 @@
     "convert-source-map": "^1.6.0",
     "lodash": "^4.17.21",
     "meteor-babel-helpers": "0.0.3",
-    "typescript": "^4.5.4"
+    "typescript": "^4.6.4"
```

I think this should be:
```suggestion
    "typescript": "~4.6.4"
```

Otherwise anywhere when this is installed without a lockfile (or without an up to date lockfile) will install the latest version which matches. Which would right now end up with 4.8.3.
I don't know how lockfiles are used in the meteor ecosystem, so this may not apply. But as I don't see a lockfile update in this PR, have you checked what the result it produces will be?